### DISCUSSION TITLE
Nordpool?

### DISCUSSION THREAD

**Discussion by JohanAlvedal** [JensenNick/huawei_solar_pees#28] [Forum]
do you know if it works with nordpool (in Sweden) or does one have to have the integration you refer to?

**Comment by JacobsenKim** [comment]
I use Nordpool (im in Denmark) just replace the sensors with youre buy and sell sensors, the sensor to replace are listet in top of huawei_solar_pees.yaml

> - 'sensor.energi_data_service' (price pr. kWh you pay for purchase/import)

>  - 'sensor.energi_data_service_sale' (price pr. kWh you receive for sale/export)

in this video i use nordpool sensor

[How 2 install PEES](https://youtu.be/uh7jRsIQwag)

**Comment by JensenNick** [comment]
@JohanAlvedal Welcome! As @JacobsenKim say, it will work as long as you replace the two price sensors. Do note that I do not have a setup that saves the changes you do in the `huawei_solar_pees.yaml` file. I suggest you save the original file in order to be able to track the changes you make. Please also note that I haven't updated the README after the release of v1.0.0. For now pay attention to setting the `_ps` sensors returning "unknown" to 0 (zero) at start (and redo that if you restart, until there is a metering). We are working on solving this "issue". You can just force charge and discharge your battery for 3-5 min (it's usually this sensor lagging).

**Reply by JensenNick** [reply]
@JohanAlvedal I've updated the README so you now have a description of what too do with the problematic economy sensors. I'd really appreciate your feedback (good or bad).

**Reply by JohanAlvedal** [reply]
Hello
I followed your guide and it worked fine. What you should perhaps clarify is that it is possible to use nordpool.

<img width="1625" alt="Skärmavbild 2024-02-07 kl  18 49 53" src="https://github.com/JensenNick/huawei_solar_pees/assets/25838439/1fee7583-0464-4db8-af6d-bcb735c4aebc">

**Reply by JensenNick** [reply]
@JohanAlvedal Thanks a lot for your feedback! Really happy to see that you managed with the few sensors giving us a bit of headache. I've added you question to the [FAQ section in the Wiki Pages](https://github.com/JensenNick/huawei_solar_pees/wiki/FAQ-%E2%80%90-Frequently-Asked-Questions#does-the-huaewi-solaer-pees-package-also-work-with-other-price-sensors-like-nordpool). I actually think the README is quite clear about this? Maybe I could ad "e.g. Nordpool" to make it even more clear for everyone?

![2024-02-07_19-16-44](https://github.com/JensenNick/huawei_solar_pees/assets/149193835/0624d5e6-3f17-4b87-87f8-78699a10b2fc)
